### PR TITLE
LPD-28802 Send email notification to user invited to a publication

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPD-28802

### CODE REVIEW

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
Screenshot 2024-07-23 at 1 39 34 PM

File: portal-impl/src/portal.properties
Comment (pei-jung) [inline portal-impl/src/portal.properties:1]: Don't forget to add the title and description for the feature flag in the language.properties

Comment (brookedalton) [reply portal-impl/src/portal.properties:1]: Oh should I change the name of the configuration setting to match the feature flag title? I noticed that most of the configurations were plural so I made it plural but I can change it

Comment (pei-jung) [reply portal-impl/src/portal.properties:1]: Are you referring to the configuration title `Publication Email Notifications`?

Comment (brookedalton) [reply portal-impl/src/portal.properties:1]: Yea the configuration page in the virtual instance scope, right now it is "Publications Email Notifications"

Comment (pei-jung) [reply portal-impl/src/portal.properties:1]: I see. Let's keep it plural to make it consistent with the other ones.

Comment (brookedalton) [reply portal-impl/src/portal.properties:1]: Should I leave the feature flag title singular?

Comment (pei-jung) [reply portal-impl/src/portal.properties:1]: Sure

**Comment by pei-jung** [comment]
Looks like there are conflicts with your language files.

**Comment by brookedalton** [comment]
Rebased and resent to https://github.com/liferay-publications/liferay-portal/pull/1317

**Pull request closed without merging**